### ISSUE TITLE
ui.open() does not work with Firefox

### ISSUE THREAD

**Issue by falkoschindler** [zauberzeug/nicegui#42]
In the following example Firefox reloads the start page.

```python
#!/usr/bin/env python3
from nicegui import ui

page = ui.page('/test', 'Test')
ui.button('Open', on_click=lambda: ui.open(page))

ui.run()
```

The console says:

```
Navigated to http://0.0.0.0:1234/test
Navigated to http://0.0.0.0:1234/ 
```

This behavior has been observed with Firefox 99 as well as 101.

**Comment by falkoschindler** [comment]
I think I found a solution: https://github.com/zauberzeug/nicegui/commit/bcc81840f678bfccf1700a4ed2e6809829b565c4

The closing socket caused a reload, which caused Firefox to ignore the redirect command. But with a short delay `ui.open` is working nicely again.

**Comment by falkoschindler** [comment]
This is solved in [v0.7.29](https://github.com/zauberzeug/nicegui/releases/tag/v0.7.29)
